### ISSUE TITLE
unix: Syscall9, SYS_FADVISE64 undefined for linux/arm

### ISSUE THREAD

**Issue by chessnokov** [golang/go#10294]
GOARCH=arm go build:
```
../../../../golang.org/x/sys/unix/zsyscall_linux_arm.go:415: undefined: Syscall9
../../../../golang.org/x/sys/unix/zsyscall_linux_arm.go:415: undefined: SYS_FADVISE64
../../../../golang.org/x/sys/unix/zsyscall_linux_arm.go:1885: Fadvise redeclared in this block
	previous declaration at ../../../../golang.org/x/sys/unix/zsyscall_linux_arm.go:414
../../../../golang.org/x/sys/unix/zsyscall_linux_arm.go:1886: undefined: Syscall9
../../../../golang.org/x/sys/unix/zsyscall_linux_arm.go:1886: undefined: SYS_FADVISE64
```
go env:
```
GOARCH="amd64"
GOBIN=""
GOCHAR="6"
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/chessnokov/workspace"
GORACE=""
GOROOT="/usr/local/go"
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0"
CXX="g++"
CGO_ENABLED="1"
```
go version:
```
go version go1.4.2 linux/amd64
```

cat /etc/issue:
```
Fedora release 21 (Twenty One)
```

**Comment by mikioh** [comment]
Merged int #10238.

**Comment by minux** [comment]
the situation for linux/arm and linux/arm64 are significantly different.
